base/shell/explorer/lang/ro-RO.rc

```diff
@@ -74,12 +74,12 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x1
 BEGIN
     GROUPBOX "Aspectul barei de activități", IDC_STATIC, 6, 6, 240, 120
     CONTROL "", IDC_TASKBARPROP_TASKBARBITMAP, "Static", SS_BITMAP | SS_SUNKEN | SS_CENTERIMAGE, 12, 18, 229, 24
-    AUTOCHECKBOX "Fi&xează bara de activități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
+    AUTOCHECKBOX "Fixează bara de acti&vități", IDC_TASKBARPROP_LOCK, 13, 45, 200, 10
     AUTOCHECKBOX "&Auto-ascunde bara de activități", IDC_TASKBARPROP_HIDE, 13, 58, 200, 10
```

Comment:
1. I do not understand the need of changing the access key on line №78, as "A" was necessary for "Aplică" button of the dialog box. When it's about dialog boxes, please avoid assigning "F", "N", and (when "Apply" button is present) also "A" access keys to other controls.
2. (Less important) I do not quite understand the reason behind moving the access key from "X" to "V" on line №77. Neither "X" got reused elsewhere in this dialog box, nor "V" was a obviously much better option for this control.

Reply:
I solved the problems. I still try to familiarize with the rules.